refarch-backend/src/main/resources/application-local.yml

```diff
@@ -3,6 +3,9 @@ server:
 spring:
   # Spring JPA
   jpa:
+    hibernate:
+      # schema is managed by Flyway, only validate
+      ddl-auto: validate
```

Comment:
I think we need to move this `ddl-auto: validate` to the `application.yml` because we also need that when running in production, right?